crates/node/src/commands/run.rs

```diff
@@ -86,13 +132,47 @@ pub fn run_node(mut cli: Cli) -> Result<()> {
             None
         }
     };
+
+    let l1_messages_worker_config = extract_l1_messages_worker_config(&cli.run.l1_messages_worker)
+        .map_err(|e| sc_cli::Error::Input(e.to_string()))?;
+    let settlement_config: Option<(SettlementLayer, PathBuf)> = match cli.run.settlement {
+        Some(SettlementLayer::Ethereum) => {
+            let config_path = data_path.join("eth-config.json");
```

Comment:
shouldn't we have a cli param to specify the config path ? Like we have for DA layer in #1287

Reply:
also, there seems to be no way to have this config created with the setup command?

Reply:
Fixed @EvolveArt comment.
@apoorvsadana I don't think the setup should create a default one, at least for now.

I also created https://github.com/keep-starknet-strange/madara/issues/1334, for someone to pick later